README.md

```diff
@@ -60,6 +60,13 @@ View the [Joy UI documentation](https://mui.com/joy-ui/getting-started/).
 
 <p>
   <a href="https://www.doit.com/?utm_source=mui.com&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="128" width="128" src="https://mui.com/static/sponsors/doit-square.svg" alt="doit" title="Management Platform for Google Cloud and AWS" loading="lazy" /></a>
+  <a href="https://kombai.com/?utm_source=MUI&utm_medium=website&utm_campaign=sponsor&utm_content=readme" rel="noopener sponsored" target="_blank">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://mui.com/static/sponsors/kombai-square-dark.svg">
+      <source media="(prefers-color-scheme: light)" srcset="https://mui.com/static/sponsors/kombai-square-light.svg">
+      <img height="128" width="128" src="https://mui.com/static/sponsors/kombai-square-light.svg" alt="kombai" title="AI agent for Frontend - turn Figma, text, or images into clean MUI code" loading="lazy" />
```

Comment:
I used dark and light logos here as well.
It doesn't show them in the preview yet: https://github.com/rluzists1/material-ui/tree/add-kombai-diamond-sponsor?tab=readme-ov-file#sponsors
But as soon as the PR is merged and deployed to production, it will look like this:

https://github.com/user-attachments/assets/4ed9f575-0004-44fb-aa8a-1bf8b8ed05ca